pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -57,30 +57,30 @@ func New(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command
 			retrieved from the update server and cached on the cluster - these are updates that are
 			known to be supported as upgrades from the current version.
 
-			Passing --to=VERSION will upgrade the cluster to one of the available updates or report
-			an error if no such version exists. The cluster will then upgrade itself and report
-			status that is available via "oc get clusterversion" and "oc describe clusterversion".
+			Passing --to=VERSION or --to-image=IMAGE will upgrade the cluster to one of the available
+			updates or report an error if no such version exists. The cluster will then upgrade
+			itself and report status that is available via "oc get clusterversion" and "oc describe
+			clusterversion".
 
-			If the desired upgrade from --to-image is not in the list of available versions, you must
-			pass --allow-explicit-upgrade to allow upgrade to proceed. If the cluster is
-			already being upgraded, or if the cluster is reporting a failure or other error, you
-			must pass --allow-upgrade-with-warnings to proceed (see note below on the implications).
-
-			If the cluster reports that the upgrade should not be performed due to a content
+			If there are no versions available, or a bug in the cluster version operator prevents
+			updates from being retrieved, --to-image may be combined with the more powerful and
+			dangerous --allow-explicit-upgrade. This instructs the cluster to upgrade to the contents
+			of the specified release image, regardless of whether that upgrade is known to be
+			supported for the current version. While rolling back to a previous micro version
+			(4.1.2 -> 4.1.1) may be safe, upgrading more than one minor version ahead (4.1 -> 4.3) or
+			downgrading one minor version (4.2 -> 4.1) is likely to cause data corruption or to
+			completely break a cluster.
+
+			If the cluster is already being upgraded, or if the cluster is reporting a failure or
+			other error, you must pass --allow-upgrade-with-warnings to proceed (see note below on
+			the implications). Avoid upgrading when the cluster is reporting errors or when another
+			upgrade is in progress unless the upgrade cannot make progress.
```

Comment:
`Avoid upgrading when the cluster is reporting errors or when another upgrade is in progress unless the upgrade cannot make progress.`  is confusing and do not communicate the context properly.  Here is my suggestion `You should understand risks from the error/warnings before considering to use --allow-upgrade-with-warnings` oe something like that. I am looking for a smaller sentence here.

Reply:
Addressed with b6bb202a8 -> 4e3339fe2:

> If the cluster is already being upgraded, or if the cluster is reporting a failure or other error, the update will not be triggered.  It is usually best to give these conditions time to resolve, or to actively work to resolve them.  But if you decide to trigger the update regardless of these concerns, use --allow-upgrade-with-warnings.